Gemfile

```diff
@@ -12,7 +9,7 @@ gem 'yajl-ruby'
 gem "rest-client"
 gem 'rake'
 gem 'mixlib-versioning', '~> 1.1.0'
-gem 'mixlib-install', :git => 'https://github.com/chef/mixlib-install.git', :branch => 'master'
+gem 'mixlib-install', '~> 1.0.11'
```

Comment:
I don't think we need a pinning here since the `Gemfile.lock` will lock down a specific version.